packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -560,43 +560,41 @@ function ensureTargetGlobalExists(
 }
 
 /**
- * An object representing the changes at a particular snapshot version for the
- * given target. This is used to facilitate storing query changelogs in the
- * targetChanges object store.
+ * An object store to store the keys of changed documents. This is used to
+ * facilitate storing document changelogs in the Remote Document Cache.
  *
  * PORTING NOTE: This is used for change propagation during multi-tab syncing
  * and not needed on iOS and Android.
  */
-export class DbTargetChange {
+export class DbDocumentChanges {
   /** Name of the IndexedDb object store.  */
-  static store = 'targetChanges';
+  static store = 'documentChanges';
 
-  /** Keys are automatically assigned via the targetId and snapshotVersion. */
-  static keyPath = ['targetId', 'snapshotVersion'];
+  /** Keys are auto-generated via the `autoGeneratedKey` property. */
+  static keyPath = 'autoGeneratedKey';
+
+  /**
+   * The auto-generated key of this entry.
+   */
```

Comment:
put this comment on one line if you want...

Reply:
Done